src/ActionModal/components/DelegationModal.vue

```diff
@@ -467,7 +472,7 @@ export default {
       `,
       /* istanbul ignore next */
       skip() {
-        return !this.address || !this.stakingDenom
+        return !this.address || !this.stakingDenom || !this.show
```

Comment:
I guess in DelegationModal the logic is missing. Would be great to use the same code though